examples/run copy.py

```diff
@@ -0,0 +1,56 @@
+import asyncio
```

Comment:
Do we still need this file?

Reply:
Deleted old working/developer's version of the file.